ui/common/css/component/_button.scss

```diff
@@ -58,6 +58,18 @@
     }
   }
 
+  &-link {
+    @extend %button-none;
+
+    color: $c-link;
+
+    &:hover,
+    &:active,
+    &:focus {
+      color: $c-link-hover;
+    }
+  }
+
```

Comment:
Not quite sure if this really should be in common but this seems like something that probably will be useful in other places as well that currently use links when they should actually be buttons (i.e. don't have a `href`).